services/apps/src/apps/clients/notifications.clj

```diff
@@ -27,6 +28,14 @@
              {:content-type :json
               :body (cheshire/encode m)}))
 
+(defn- guarded-send-notification
```

Comment:
I also want to spend a little time separating the notification formatting from the sending. That namespace seems to be getting a little cluttered.